pkg/operator/ceph/file/subvolumegroup/controller.go

```diff
@@ -255,12 +254,29 @@ func (r *ReconcileCephFilesystemSubVolumeGroup) reconcile(request reconcile.Requ
 	if err != nil {
 		return reconcile.Result{}, errors.Wrap(err, "failed to save cluster config")
 	}
+
+	err = r.pinSubvolumeGroup(cephFilesystemSubVolumeGroup)
```

Comment:
Looks like every subvolumegroup will be pinned, whether new or upgraded. @batrick Should we allow opting out of the pinning?

Reply:
There should be a config to disable pinning, yes. The default should be distributed=1.

Reply:
To disable pinning is it set to 0? Or what are the possible values? If the only two values are 0 and 1, we could we treat it as a bool in the Rook CRD.

Reply:
Yes, just 0 or 1. A bool is appropriate.

Reply:
maybe pin setting too.
```
pinning:
  pinType: distributed | random | export
  pinSetting: 1
  enabled: true | false

```

Reply:
@travisn

Reply:
If `pinSetting` is still needed, I don't understand why.

Reply:
By default, we can set it to true.

Reply:
PinSetting or setting can have these values that user can set

```
It depends on which pin type you're setting. export can be <-1 (disabled), 0-256>. distributed <0, 1>, and random [0.0, 1.0].
```